vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
```

Comment:
**praise**: Transitioning to TypeScript in the Vite configuration file is a great move for type safety.